### ISSUE TITLE
New Orbit Icon customization tanks FPS in Planetarium Camera

### ISSUE THREAD

**Issue by Poodmund** [Kopernicus/Kopernicus#309]
The new custom Orbit Icon feature introduced in 1.4.5-3 seems to tank the FPS when the Planetarium camera is active. 

Reproduction steps:
- Vanilla KSP
- Go to Tracking Station
- View FPS
---
- Vanilla KSP install + Kopernicus 1.4.5-4 + Dependencies
- Go to Tracking Station
- View FPS

See the difference. These are the values I pulled from my tests:
```
KSP 1.4.5 Stock                                         165fps (monitor limit)
KSP 1.4.5 + Kopernicus 1.4.5-4                          80fps
KSP 1.4.5 + Kopernicus 1.4.5-3                          85fps
KSP 1.4.5 + Kopernicus 1.4.5-2                          165fps (monitor limit)
KSP 1.4.5 + Kopernicus 1.4.5-4 w/ Orbit Icons Removed   165fps (monitor limit)
```
It seems that the custom Orbit Icon script added to Kopernicus.RuntimeUtility.cs is the cause as when it is removed and recompiled, the issue is not present. The code in question:
```
// Apply orbit icon customization
foreach (MapNode node in Resources.FindObjectsOfTypeAll<MapNode>())
{
    if (node.mapObject != null && node.mapObject.celestialBody != null && node.mapObject.celestialBody.Has("iconTexture"))
    {
        Texture2D texture = node.mapObject.celestialBody.Get<Texture2D>("iconTexture");
        node.SetIcon(Sprite.Create(texture,
            new Rect(0, 0, texture.width, texture.height),
            new Vector2(0.5f, 0.5f), 100, 1, SpriteMeshType.Tight,
            Vector4.zero));
    }
}
```
Referenced from this post in the Kopernicus thread: https://forum.kerbalspaceprogram.com/index.php?/topic/140580-145-4-131-13-kopernicus-kittopiatech/&do=findComment&comment=3443919

**Comment by marr75** [comment]
Good code-sleuthing, @Poodmund. Looking it over, I'd guess that iterating through all resources every time is going to be the cause of the slowdown so I'm trying just the CBs in FlightGlobals. Two problems in testing this: the slowdown wasn't particularly noticeable for me and I don't know of any planet packs released today that use the orbital icon customization. If anyone can try out the replaced loop below and report on whether their performance improved, that would be helpful. If anyone can tell me where to find planet pack with orbital icons, that would be doubly helpful.

```csharp
// Apply orbit icon customization
foreach (CelestialBody body in FlightGlobals.Bodies)
{
    if (body.Has("iconTexture") && body.MapObject != null && body.MapObject.uiNode != null)
    {
        MapNode node = body.MapObject.uiNode;
        Texture2D texture = node.mapObject.celestialBody.Get<Texture2D>("iconTexture");
        Sprite sprite = Sprite.Create(
            texture,
            new Rect(0, 0, texture.width, texture.height),
            new Vector2(0.5f, 0.5f),
            100,
            1,
            SpriteMeshType.Tight,
            Vector4.zero
        );
        node.SetIcon(sprite);
    }
}
```

**Comment by marr75** [comment]
Dear diary, today Poodmund, of Skybox and OPMVO fame, QA'ed one of my fixes. :-)

**Comment by StollD** [comment]
Should be resolved by merging #310 

Thanks @marr75 (and @Poodmund for testing)
